templates/CRM/Event/Form/Selector.tpl

```diff
@@ -60,15 +60,14 @@
     <td class="right nowrap crm-participant-participant_fee_amount">{if array_key_exists('participant_fee_amount', $row)}{$row.participant_fee_amount|crmMoney:$row.participant_fee_currency}{/if}</td>
     <td class="crm-participant-participant_register_date">{$row.participant_register_date|truncate:10:''|crmDate}</td>
     <td class="crm-participant-event_start_date">{$row.event_start_date|truncate:10:''|crmDate}
-        {if $row.event_end_date && $row.event_end_date|crmDate:"%Y%m%d" NEQ $row.event_start_date|crmDate:"%Y%m%d"}
-            <br/>- {$row.event_end_date|truncate:10:''|crmDate}
-        {/if}
-   </td>
```

Comment:
Is removing this `</td>` right? It looks correct as-is.

Reply:
@demeritcowboy I just fixed the indentation above & then I could see what it goes with so it's back now